elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/AggregationStoreTest.java

```diff
@@ -35,24 +41,60 @@
                     + "\t\t(1,100,'Foo'),"
                     + "\t\t(2,150,'Bar');"
     })
-    public void jsonApiGetTest(@Autowired MeterRegistry metrics) {
-        when()
-                .get("/json/stats?fields[stats]=measure")
-                .then()
-                .body(equalTo(
-                        data(
-                                resource(
-                                        type("stats"),
-                                        id("0"),
-                                        attributes(
-                                                attr("measure", 250)
-                                        )
+    public void jsonApiGetTestNoCache(@Autowired MeterRegistry metrics) {
+
+        Map<String, String> requestHeaders = new HashMap<>();
+        requestHeaders.put("bypassCache", "true");
+
+        given().headers(requestHeaders)
+        .get("/json/stats?fields[stats]=measure")
+        .then()
+        .body(equalTo(
+                data(
+                        resource(
+                                type("stats"),
+                                id("0"),
+                                attributes(
+                                        attr("measure", 250)
                                 )
-                        ).toJSON())
-                )
-                .statusCode(HttpStatus.SC_OK);
+                        )
+                ).toJSON())
+        )
+        .statusCode(HttpStatus.SC_OK);
+        assertFalse(metrics
+                .get("cache.gets")
+                .tags("cache", "elideQueryCache", "result", "miss")
+                .functionCounter().count() > 0);
+    }
+
+    @Test
+    @Sql(statements = {
+            "DROP TABLE Stats IF EXISTS;",
+            "CREATE TABLE Stats(id int, measure int, dimension VARCHAR(255));",
+            "INSERT INTO Stats (id, measure, dimension) VALUES\n"
+                    + "\t\t(1,100,'Foo'),"
+                    + "\t\t(2,150,'Bar');"
+    })
+    public void jsonApiGetTestWithCache(@Autowired MeterRegistry metrics) {
 
-        // query cache was active and publishing metrics
+        Map<String, String> requestHeaders = new HashMap<>();
+        requestHeaders.put("bypassCache", "false");
+
+        given().headers(requestHeaders)
+        .get("/json/stats?fields[stats]=measure")
+        .then()
+        .body(equalTo(
+                data(
+                        resource(
+                                type("stats"),
+                                id("0"),
+                                attributes(
+                                        attr("measure", 250)
+                                )
+                        )
+                ).toJSON())
+        )
+        .statusCode(HttpStatus.SC_OK);
```

Comment:
This test is the same as the prior.  Maybe we should look at the cache metrics for something different between the tests?